Development/src/dataProvider.js

```diff
@@ -254,10 +263,11 @@ const convertDataProviderRequestToHTTP = (type, resource, params) => {
             const legs = get(params, 'data.$staged.transport_params').length;
             for (let i = 0; i < legs; i++) {
                 patchData.transport_params.push({});
+                patchData.transport_params.push({});
```

Comment:
Hmm. Why?

Reply:
No clue how that made it in, don't remember making the change myself. Removed